### ISSUE TITLE
benchmark re10k-4dim

### ISSUE THREAD

**Issue by lee-wanhee** [Stability-AI/stable-virtual-camera#2]
Thank you so much for the amazing work!

Correct me if I am wrong, but I noticed that the re10k-4dim scenes appear to be part of the training set in RealEstate10K rather than the test set.

https://google.github.io/realestate10k/download.html 

For example, the first three scenes in the re10k-4dim, 00a861d699fb7797, 00b9a7963f9bd9c6, and 0c7cadc659ac1393, are all in the training set of re10k.

This seems to differ from what is stated in the paper. Could you take a look and confirm?

<img width="449" alt="Image" src="https://github.com/user-attachments/assets/82ce55d4-e70e-46c5-a41f-bb73eb004793" />

**Comment by hangg-sai** [comment]
Nice catch! We will verify this and get back to you--we follow what is provided from 4DiM authors, it's possible that they didn't follow the official training/testing split.

**Comment by hangg-sai** [comment]
Hi @lee-wanhee, you are right on this one, we can verify this as well. This is totally our hindsight, we will do the following:
(1) Remove this comparison from our Table 2;
(2) Mention it in the text that 4DiM is training on its own train/test split which is different from the standard train/test split that we follow;
(3) Keep the visual comparison in the Figure 6 as that is still a out-of-distribution fair comparison! 

Thanks again for helping us locating this issue 🙌
